### ISSUE TITLE
update package on conda-forge

### ISSUE THREAD

**Issue by prjemian** [kapouille/mongoquery#26]
The latest version on [conda-forge](https://anaconda.org/conda-forge/mongoquery) is 1.3.5.  Can you update to a more recent version?  At least 1.3.6 is listed in requirements for [apstools](https://github.com/BCDA-APS/apstools), part of [bluesky](https://blueskyproject.io/).

**Comment by prjemian** [comment]
Here's the minimum [1.3.6](https://github.com/conda-forge/mongoquery-feedstock/pull/8) version needed, waiting on approvals.

@tacaswell , @CJ-Wright, @ericdill, @licode - Any of you care to review this 2+ year-old PR?

**Comment by prjemian** [comment]
Passing review not required, just merge by authorized GH user.

**Comment by tacaswell** [comment]
I think this issue should be closed, we can handle this down-stream.

**Comment by prjemian** [comment]
How can I release apstools on conda-forge with a non-cf requirement?

**Comment by tacaswell** [comment]
No, I mean this is a problem for us and conda-forge, we should not be bringing the upstream project into this.

A core tenant of CF when it started is that upstream maintainers who do not want to work on conda/conda-forge do not have to.  This is not their problem and we should not be discussion this in the mongoquery's issues.

**Comment by tacaswell** [comment]
and I got the 1.4.2 PR merged.
